src/app.ts

```diff
@@ -13,14 +13,49 @@ interface TemperatureSummary {
   average: number
 }
 
+
+const redingMap: { [key: string]: { [key: string]: number[] } } = {}; 
```

Comment:
```suggestion
const readingMap: { [key: string]: { [key: string]: number[] } } = {}; 
```

Reply:
you have a typo for readings variables, I would also prefer use 
temperatureReadings instead of readingMap

Reply:
- I will rename the variables ✅
- Yes you can declare the type before using the variable, this way you can quickly see what expects, (strings and numbers)